openwisp_radius/api/freeradius_views.py

```diff
@@ -525,6 +533,39 @@ def send_radius_accounting_signal(self, accounting_data):
             view=self,
         )
 
+    def _handle_accounting_on(self, data):
+        """
+        Handle Accounting-On packets by triggering a background task
+        to close stale sessions for the NAS.
+
+        When a NAS (Network Access Server) sends an Accounting-On packet,
+        it indicates that the NAS has restarted or experienced a power failure.
+        All active sessions on that NAS should be considered stale and closed.
+        """
+        nas_ip_address = data.get("nas_ip_address")
+        if not nas_ip_address:
+            logger.warning("Accounting-On packet received without nas_ip_address")
+            return
+
+        organization_id = self.request.auth
+        if not organization_id:
+            logger.warning("Accounting-On packet received without valid organization")
```

Comment:
Well, I am not sure about this. I believe in this case we can just ignore the organization: if a NAS has been cold rebooted, we can just flag any accounting session related to it to terminated, regardless of the org, therefore we could avoid checking and requiring the org. I don't see downsides in this. The pro is that it simplifies the implementation.